src/cljs/athens/views/blocks/textarea_keydown.cljs

```diff
@@ -392,16 +392,19 @@
         selected-items                 @(subscribe [::select-subs/items])
         editing-uid                    @(subscribe [:editing/uid])
         current-root-uid               @(subscribe [:current-route/uid])
-        [editing-uid embed-id]         (db/uid-and-embed-id editing-uid)]
+        [editing-uid embed-id]         (db/uid-and-embed-id editing-uid)
+        local-string                   (:string/local @_state)]
```

Comment:
don't leave `_` in `_state` if we're using it now.
`_` at the beginning means not used.